### PR TITLE
SPARK-13034[ML]:PySpark ml.classification support export/import

### PR DESCRIPTION
**Pull request by wangmiao1981**
I added Import and Export for Logisticregression and Naive Bayes

Test ./python/run-tests --python-executables=python2.7 --modules=pyspark-ml
Result:
Running PySpark tests. Output is in /Users/mwang/bugfix/spark_bug_fix/python/unit-tests.log
Will test against the following Python executables: ['python2.7']
Will test the following Python modules: ['pyspark-ml']
Finished test(python2.7): pyspark.ml.feature (26s)
Finished test(python2.7): pyspark.ml.recommendation (27s)
Finished test(python2.7): pyspark.ml.classification (27s)
Finished test(python2.7): pyspark.ml.clustering (27s)
Finished test(python2.7): pyspark.ml.evaluation (12s)
Finished test(python2.7): pyspark.ml.tuning (17s)
Finished test(python2.7): pyspark.ml.regression (21s)
Finished test(python2.7): pyspark.ml.tests (21s)
Tests passed in 48 seconds

### CODE REVIEW

**Comment by srowen** [comment]
@wangmiao1981  can you fix the title? and maybe squash your commits once upfront here. There are 32 already.

**Comment by wangmiao1981** [comment]
@srowen I added the title in the pull request. Sorry for causing the confusion here. I only made changes in one python file. All other changes are merged from master to my private branch. I don't know why they are included in the PULL request. I will update it and clean it.

**Comment by jkbradley** [comment]
@GayathriMurali @wangmiao1981 It will be great to add persistence for LogisticRegression and NaiveBayes.  The other algorithms will need to wait since they do not yet have Scala support for persistence.  Could you please reconcile your 2 PRs?  It would be great to close one and have both of you collaborate on 1 PR.  Thanks!

**Comment by GayathriMurali** [comment]
@wangmiao1981 I closed my PR. I have made the changes to add persistence to only LogisticRegression and Naive Bayes. Would you like to review? We could then send one PR.

**Comment by jkbradley** [comment]
Thanks both for collaborating!

@wangmiao1981 If you want to get rid of the commits merged from master, then you can do ```git rebase upstream/master```.  Feel free to do that now to clean up the commits.  In general, though, try not to rebase a PR unless there are merge conflicts because it makes the PR history harder to understand.

Can you please revert the changes to the files in ```bin/```?  It looks like you changed the permissions and committed that change.

I'll take a look at the other parts now.

**Comment by jkbradley** [comment]
Also, to "fix the title," you'll need to put square brackets around the JIRA: "[SPARK-13034]" and probably remove the colon ":"

File: sql/hive/pom.xml [hunk 1]
```diff
@@ -38,6 +38,11 @@
   <dependencies>
     <!-- Added for Hive Parquet SerDe -->
     <dependency>
+    <groupId>org.scala-lang</groupId>
```
Comment (jkbradley) [inline sql/hive/pom.xml]: Please revert the change to this file; it should not be needed.

**Comment by jkbradley** [comment]
It looks like [https://github.com/apache/spark/pull/11707] is closer to being merged now, so let's go with it instead.  @wangmiao1981 Could you please close this PR?

**Comment by wangmiao1981** [comment]
close this one as it has been merged with 11707.
